Bottom: b5c8f15fdc658a09e39eaba0eaee2d7669312a21
Top:    266d17396a6c44ce2322dbd6fb5081e8acaf09fc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 02:56:12 +0100

macro.h: Remove unused X11 macros


---

diff --git a/src/macro.h b/src/macro.h
index 484f284..7cba5ac 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -149,20 +149,6 @@ extern int mem_any_set (unsigned char *, int);
 #define	NAMEONPCB_TEXT(e)	((e)->Name[NAMEONPCB_INDEX])
 #define	VALUE_TEXT(e)		((e)->Name[VALUE_INDEX])
 
-/* ----------------------------------------------------------------------
- * checks for correct X values
- */
-#define	VALID_PIXMAP(p)	((p) != BadValue && \
-	(p) != BadAlloc && \
-	(p) != BadDrawable)
-
-#define	VALID_GC(p)	((p) != BadValue && \
-	(p) != BadAlloc && \
-	(p) != BadDrawable && \
-	(p) != BadFont && \
-	(p) != BadMatch && \
-	(p) != BadPixmap)
-
 /* ---------------------------------------------------------------------------
  *  Determines if text is actually visible
 #define TEXT_IS_VISIBLE(b, l, t)					\
